### PR TITLE
Add copy buttons to code snippets

### PR DESCRIPTION
**Pull request by Vekat**
An attempt to solve issue #10 with some JavaScript.

Is this close to what was expected?

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ for (var i = 0; i < btns.length; i++) {
 }
 
 function showTooltip(elem, msg) {
-    elem.setAttribute('class', 'btn tooltipped tooltipped-s');
+    elem.classList.add('tooltipped');
+    elem.classList.add('tooltipped-s');
```
Comment (lipis) [inline assets/scripts/main.js]: Not supported in IE9: http://caniuse.com/#feat=classlist

Comment (Vekat) [inline assets/scripts/main.js]: I see.

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/zenorocha/clipboard.js/compare/dbe4b1f~3...dbe4b1f) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Hey @Vekat,

I've made a bunch of changes but you've got the essence. It's merged and live ;)

Thanks for contributing and sorry for taking so long to review it.

**Pull request closed without merging**